website/servers/server.php

```diff
@@ -82,13 +82,77 @@
 			die("invalid server location");
 		}
 
-		$PokemonServers[$id]['server'] = $server;
-		$PokemonServers[$id]['port'] = $port;
-	}
+		$oldServer = $PokemonServers[$id]['server'];
+		$oldPort = $PokemonServers[$id]['port'];
 
-	$owners = explode(',', $_POST['owner']);
-	foreach ($owners as $i=>$owner) $owners[$i] = $users->userid($owner);
-	$PokemonServers[$id]['owner'] = implode(',',$owners);
+		$cannotUpdate = false;
+		if (isset($serverinfo['bannedhosts']) && trim($server) !== trim($oldServer)) {
+			foreach ($serverinfo['bannedhosts'] as $bannedhost) {
+				$bannedhost = trim($bannedhost);
```

Comment:
Why isn't `$serverinfo['bannedhosts']` pre-trimmed?

Reply:
It is, this was just a force-of-habit sanity check.